.gitignore

```
@@ -18,6 +18,7 @@
 # Executables produced by temporal repo
 /temporal-*
 /tctl*
+/tdbg*
```

```suggestion
/tdbg
```
and can you remove line 20.